Packs/Stealthwatch_Cloud/Integrations/Stealthwatch_Cloud/Stealthwatch_Cloud.py

```diff
@@ -558,7 +558,7 @@ def fetch_incidents():
         demisto.results(list_sessions_command())
     elif demisto.command() == 'fetch-incidents':
         demisto.results(fetch_incidents())
-except Exception, e:
+except Exception as e:
```

Comment:
This is python 2 integration, not python 3

Reply:
This syntax works in BOTH.